zipkin-server/src/main/java/zipkin2/server/internal/throttle/ThrottledCall.java

```diff
@@ -38,11 +38,21 @@
  * @see ThrottledStorageComponent
  */
 final class ThrottledCall extends Call.Base<Void> {
+  static final Callback<Void> NOOP_CALLBACK = new Callback<Void>() {
+    @Override public void onSuccess(Void value) {
+    }
+
+    @Override public void onError(Throwable t) {
+    }
+  };
+
   final Call<Void> delegate;
   final Executor executor;
   final Limiter<Void> limiter;
   final LimiterMetrics limiterMetrics;
   final Predicate<Throwable> isOverCapacity;
+  final CountDownLatch latch = new CountDownLatch(1);
```

Comment:
Would you be opposed to moving this instantiation into the existing constructor?  By leaving it up here you litterally end up with two constructors in your class which can unnecessarily increase class size and slown down instantiation times.  Unless the HotSpot optimizer can detect and fix this of course ;)